docs/guide.rst

```diff
@@ -86,3 +86,141 @@ options::
 
 Go on and explore. Don't be shy and ask, ``--help`` is eager to
 answer all your questions ;-)
+
+
+Under The Hood
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now let's have a brief look under the hood. Similarly as with
+``git``, there is a special ``.fmf`` directory which marks root of
+the fmf metadata tree. Use the ``init`` command to initialize it::
+
+    tmt init
+
+Do not forget to include this special ``.fmf`` directory in your
+commits, it is essential for building the fmf tree structure which
+is created from all ``*.fmf`` files discovered under the fmf root.
+
+
+Plans
+------------------------------------------------------------------
+
+As we've seen above, in order to enable a simple test the
+following plan is just enough::
+
+    execute:
+        script: foo --version
+
+Store these two lines in a ``*.fmf`` file and that's it. Name and
+location of the file is completely up to you, plans are recognized
+by the ``execute`` key which is required. Once the newly created
```

Comment:
This is great info even for new contributors, I found it a bit confusing at first how stories, tests and plans are differentiated since they are all just fmf files. It's great that it's finally documented here :)

Reply:
Good to hear. Thanks for the feedback!